js/advanced-search.js

```diff
@@ -11,6 +11,48 @@
  */
 var Bugzilla = Bugzilla || {}; // eslint-disable-line no-var
 
+Bugzilla.AdvancedSearch = {};
+
+/**
+ * Implement features in the Search By Change History section.
+ */
+Bugzilla.AdvancedSearch.HistoryFilter = class HistoryFilter {
+  /**
+   * Initialize a new HistoryFilter instance.
+   */
+  constructor() {
+    this.$chfield = document.querySelector('#chfield');
+    this.$chfieldfrom = document.querySelector('#chfieldfrom');
+    this.$chfieldfrom_button = document.querySelector('#chfieldfrom + button');
+    this.$chfieldto = document.querySelector('#chfieldto');
+    this.$chfieldto_button = document.querySelector('#chfieldto + button');
+
+    this.$chfieldfrom.addEventListener('input', event => this.on_date_change(event.target));
+    this.$chfieldto.addEventListener('input', event => this.on_date_change(event.target));
```

Comment:
s/event.target/event/

Reply:
Aha, why I didn’t notice this… Fixing now.